src/apprt/gtk/Window.zig

```diff
@@ -1007,128 +1010,87 @@ fn gtkActionNewWindow(
     _: ?*glib.Variant,
     self: *Window,
 ) callconv(.C) void {
```

Comment:
Since we're messing around in this area Zig 0.14 is deprecating `callconv(.C)` in favor of `callconv(.c)`.